partner_communication_switzerland/models/mail_tracking_event.py

```diff
@@ -61,23 +61,33 @@ def send_mails_to_partner_and_staff(self, tracking_email, metadata,
                                  content_subtype='plaintext')
             partner.write(to_write)
 
+    def _remove_address_from_sendgrid_bounce_list(self, tracking_email):
+        tracking_email.partner_id.message_post(
+            _('Partner remove from sendgrid bounced e-mails'))
```

Comment:
Instead of indicating that the partner was removed from the list, it's more useful to indicate which e-mail was bounced and not delivered to the sponsor:
```suggestion
            _('The email was bounced and not delivered to the partner'),
            tracking_email.name
            )
```